requirements.txt

```diff
@@ -32,3 +32,4 @@ uvicorn==0.30.6
 wrapt==1.16.0
 yara-python==4.5.1
 yaramod==3.23.0
+alembic_postgresql_enum
```

Comment:
I am positive this can be done without any additional dependencies.

https://stackoverflow.com/questions/63461381/how-to-use-an-existing-sqlalchemy-enum-in-an-alembic-migration-postgres
https://stackoverflow.com/questions/37848815/sqlalchemy-postgresql-enum-does-not-create-type-on-db-migrate

suggest a solution with postgresql.ENUM (I remember doing something like that too).

Reply:
as far as I can see it's not used and can be removed? We definitely don't want a new dependency.